From a98b194f899795d4cb6a1be0906f91e99b190094 Mon Sep 17 00:00:00 2001
From: Nicolin Chen <b42378@freescale.com>
Date: Tue, 5 Nov 2013 19:23:58 +0800
Subject: [PATCH 0287/5242] MLK-11418 misc: sram: Set default alignment to
 4Kbytes

commit  89d38957981be1bec834af90a7920e4b1e3e343e from
https://source.codeaurora.org/external/imx/linux-imx.git

cherry-pick below patch from v3.14.y:
ENGR00286273-2 misc: sram: Set default alignment to 4Kbytes

As Kernel 3.0.35 does, we set the default iram alignment to 4Kbytes,
although it would waste few memory space.

We here try to do this as an expediency because currently we couldn't
find a perfect solution for 4Kbytes alignment requirement from SDMA.

Signed-off-by: Nicolin Chen <b42378@freescale.com>
(cherry picked from commit cc35bf5bd3472efca29a0858c0ad468a63c60950)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/misc/sram.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/misc/sram.c b/drivers/misc/sram.c
index 74b183b..becd12e 100644
--- a/drivers/misc/sram.c
+++ b/drivers/misc/sram.c
@@ -33,7 +33,7 @@
 
 #include "sram.h"
 
-#define SRAM_GRANULARITY	32
+#define SRAM_GRANULARITY	4096
 
 static ssize_t sram_read(struct file *filp, struct kobject *kobj,
 			 struct bin_attribute *attr,
-- 
1.7.9.5

